UIComponents/UIComponents/Helper/CLLocationHelper.swift

```diff
@@ -25,10 +25,9 @@ class CLLocationHelper: NSObject {
 
     // iPhone location service check
     func checkLocationServices(locationCompletionHandler: ((CLLocation?) -> Void)?) {
-        DispatchQueue.global().async { [weak self] in
+        DispatchQueue.global().async {
             if CLLocationManager.locationServicesEnabled() {
-                self?.checkLocationAuth()
```

Comment:
why we have remove weak self in this closure, will it not create strong reference?